src/controllers/usersController.js

```diff
@@ -0,0 +1,85 @@
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import successRes from '../helpers/successHandler';
+import errorRes from '../helpers/errorHandler';
+import Models from '../database/models';
+import userValidationSchema from '../validators/userValidator';
+
+const { User } = Models;
+
+export const register = async (req, res) => {
+  try {
+    const validateUser = userValidationSchema.validate(req.body);
+    const {
+      firstName,
+      lastName,
+      email,
+      nationalId,
+      phone,
+      password,
+      role,
+    } = req.body;
+    if (validateUser.error) {
+      console.log(validateUser.error);
```

Comment:
`console.log()` is for debugging. So once you are sure that what you want is what you are getting, it is a good practice to remove it from your code